firefox-ios/WidgetKit/DownloadManager/DownloadLiveActivity.swift

```diff
@@ -53,12 +55,60 @@ struct DownloadLiveActivityAttributes: ActivityAttributes {
 @available(iOS 16.2, *)
 struct DownloadLiveActivity: Widget {
     var body: some WidgetConfiguration {
-        ActivityConfiguration(for: DownloadLiveActivityAttributes.self) { _ in
+        ActivityConfiguration(for: DownloadLiveActivityAttributes.self) { LiveDownload in
             // Using Rectangle instead of EmptyView because the hitbox
             // of the empty view is too small (likely non existent),
             // meaning we'd never be redirected to the downloads panel
-            Rectangle()
-                .widgetURL(URL(string: URL.mozInternalScheme + "://deep-link?url=/homepanel/downloads"))
+            ZStack {
+                Rectangle()
+                    .widgetURL(URL(string: URL.mozInternalScheme + "://deep-link?url=/homepanel/downloads"))
+                    .foregroundStyle(LinearGradient(
+                        gradient: Gradient(colors: [Color("searchButtonColorTwo"), Color("searchButtonColorOne")]),
+                        startPoint: .topLeading,
+                        endPoint: .bottomTrailing
+                    ))
+                HStack(spacing: 16) {
+                    ZStack {
+                        Image("faviconFox")
+                            .resizable()
+                            .scaledToFit()
+                            .frame(width: 40, height: 40)
+                    }
+                    VStack(alignment: .leading, spacing: 4) {
+                        Text("Downloading \(LiveDownload.state.downloads[0].fileName)")
```

Comment:
For all user facing text, these strings have been localized. We should instead use the localized versions of these strings so that they are available to the user in different languages. See more here: https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings. For the list of localized strings, take a look at Raine's pr -- https://github.com/mozilla-mobile/firefox-ios/pull/24804/files

Reply:
@ih-codes I don't 100% remember, but I think there was mention of special considerations when using localized strings with swiftui. Can you confirm if there is anything else we need to take into consideration related to using localized strings before merging?